Makefile

```diff
@@ -119,29 +123,23 @@ setup-bare: clean e2e.namespace
 	. ./scripts/package_release.sh 1.0.0 test/e2e/resources test/e2e/e2e-bare-values.yaml
 	. ./scripts/install_bare.sh $(shell cat ./e2e.namespace) test/e2e/resources
 
-GINKGO_OPTS := -flakeAttempts 3 -randomizeAllSpecs -v --timeout 120m
+E2E_NODES ?= 1
+E2E_FLAKE_ATTEMPTS ?= 1
+E2E_TIMEOUT ?= 90m
+E2E_OPTS ?= $(if $(E2E_SEED),-seed '$(E2E_SEED)') $(if $(TEST),-focus '$(TEST)') -flakeAttempts $(E2E_FLAKE_ATTEMPTS) -nodes $(E2E_NODES) -timeout $(E2E_TIMEOUT) -randomizeAllSpecs -v
+E2E_INSTALL_NS ?= operator-lifecycle-manager
+E2E_TEST_NS ?= operators
```

Comment:
Reminder: we need to set the downstream-specific namespaces in the root `Makefile` downstream before invoking this target; e.g. `E2E_INSTALL_NS=openshift-operator-lifecycle-manager E2E_TEST_NS=openshift-operators  $(MAKE) staging/operator-lifecycle-manager/Makefile`